pulser/simulation/simresults.py

```diff
@@ -15,15 +15,20 @@
 import qutip
 import numpy as np
 
+from collections import Counter
+from abc import ABC, abstractmethod
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports), imports from the standard library should come before imports from third-party libraries.